github/ci/prow-deploy/kustom/base/manifests/local/prow-debug-rbac.yaml

```diff
@@ -0,0 +1,41 @@
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: prow-debug
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: prow-debug
+rules:
+- apiGroups:
+  - "prow.k8s.io"
+  resources:
+  - "prowjobs"
+  verbs:
+  - "get"
+  - "list"
+  - "create"
```

Comment:
Do we want to allow creating prowjobs? Isn't it equivalent to creating an arbitrary pod?

Reply:
No, [prowjobs](https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/kustom/base/manifests/test_infra/current/prowjob-crd/prowjob_customresourcedefinition.yaml) are controlled by prow-controller-manager, who then manages the job pods that are created. I thought that we might want to enable people to create them to be able to debug. WDYT?

Reply:
Besides that, prowjobs can only be created on ibm-prow-jobs cluster.

Reply:
I would agree with @xpivarc  - prowjob yamls can be manipulated to run almost anything as privileged on the cluster. I would remove `create` for now and if we find it is needed add it later? 

Other than that it looks very good to me.

Reply:
Thanks, makes sense.

Reply:
I apologize for my response time.

Reply:
Done.